scripts/scripts.js

```diff
@@ -560,3 +561,15 @@ export async function loadConsentManager() {
   ]);
   window.dispatchEvent(new CustomEvent('consentmanager'));
 }
+
+/**
+ * Formats a unix timestamp into a date string.
+ * Example: 1609459200 -> January 1, 2021
+ * @param {*} unixTimestamp
+ * @returns {string} The formatted date string
+ */
+export function formatDateFromUnixTimestamp(unixTimestamp) {
```

Comment:
i think we should maintain consistency with date format across the whole website.
For that u can change logic of [getFormattedDate function](https://github.com/hlxsites/sunstar/blob/main/scripts/lib-franklin.js#L693) and also create a wrapper for this function which takes timestamp as input.